legal-api/tests/unit/services/test_authorization.py

```diff
@@ -472,18 +472,18 @@ def mock_auth(one, two):  # pylint: disable=unused-argument; mocks of library me
          ['LLC'], 'staff', [STAFF_ROLE], False),
 
         ('staff_active_allowed', Business.State.ACTIVE, 'dissolution', 'voluntary',
-         ['CP', 'BC', 'BEN', 'CC', 'ULC'], 'staff', [STAFF_ROLE], True),
+         ['CP', 'BC', 'BEN', 'CC', 'ULC'], 'staff', [STAFF_ROLE], False),
```

Comment:
I would expect this to return `True` as I believe this test is testing the happy path of `is_allowed`.  Please find out why it's not returning `True` and fix it.

We should actually add `SP, GP` to `['CP', 'BC', 'BEN', 'CC', 'ULC']`.

Reply:
`dissolution.voluntary` is blocked by `Business.NOT_IN_GOOD_STANDING`.
But, I am not sure how to create a business that is `GOOD_STANDING`.

Reply:
I'm assuming the business is not in good standing because of the founding date.  You can update the `create_business` function in `test_authorization.py` to call `factory_business` with an explicit value for `founding_date`.  A founding of the current date would probably work.